lib/guardian/error_handler.ex

```diff
@@ -0,0 +1,40 @@
+if Code.ensure_loaded?(Plug) do
+  defmodule Guardian.ErrorHandler do
```

Comment:
Since this is related to `Plug` I would change the scope to be under `Guardian.Plug` like `Guardian.Plug. ErrorHandler`